apps/web/actions/albums/show.rb

```diff
@@ -4,18 +4,14 @@ module Web
   module Actions
     module Albums
       class Show < Web::Action
-        def initialize(albums: AlbumRepository.new)
-          @albums = albums
-        end
+        include Deps[repo: "repositories.album_repository"]
```

Comment:
This shows the new `Deps` auto-injection mixin (previously known as `Import`) working nicely for action classes.

(Will explain the naming in the main PR to hanami/hanami)